packages/core/tests/e2e/next-server/app/streamable/actions.jsx

```diff
@@ -7,22 +7,35 @@ function sleep(ms = 0) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+export async function streamableUIAppend() {
+  const streamable = createStreamableUI();
+  (async () => {
+    await sleep(10);
+    streamable.update(<p>foo</p>);
+    await sleep(10);
+    streamable.append(<p>bar</p>);
+    await sleep(10);
+    streamable.done();
+  })();
+  return streamable.value;
+}
+
 export async function streamableUI() {
   const streamable = createStreamableUI();
   (async () => {
-    await sleep();
+    await sleep(10);
```

Comment:
how stable is the test with this low timeout?

Reply:
Quite stable. Ran this many times on local/CI and the result is consistent. Here it could be `sleep(0)` and it will still work, but decided to add a small delay just in case one layer (Next.js, RSC, React, browser) batches the chunks.